### PR TITLE
Documentation changes to support BitBucket setup steps

### PR DESCRIPTION
**Pull request by archanaganesan**
- Documentation changes to support BitBucket setup steps
- Release notes for v1.2

### CODE REVIEW

File: docs/source/install.rst [hunk 1]
```diff
@@ -206,6 +206,24 @@ Additionally, you may want to add env vars for the Jenkins container, and the JV
 Additional Configuration
 --------------------------------------
 
+**Create Initial BitBucket Account**
```
Comment (lonico) [inline docs/source/install.rst]: spelling: Bitbucket (here and below)

File: docs/source/install.rst [hunk 2]
```diff
@@ -206,6 +206,24 @@ Additionally, you may want to add env vars for the Jenkins container, and the JV
 Additional Configuration
 --------------------------------------
 
+**Create Initial BitBucket Account**
+
+
+    An initial account has to be created on BitBucket before starting to use it.
+
+    To create an account on Gitlab, visit the following URL and sign up.
```
Comment (lonico) [inline docs/source/install.rst]: Gitlab -> Bitbucket

File: docs/source/install.rst [hunk 3]
```diff
@@ -206,6 +206,24 @@ Additionally, you may want to add env vars for the Jenkins container, and the JV
...
+    Please note that the account credentials for the account should be root (username) and root_devopsatscale (password)
+
+    .. code :: shell
+
+        http://<<$NODE_IP>>:<<BitBucket_service_port>>
+
+
+    .. figure:: images/bitbucket.png
+        :width: 100%
+        :alt: GitLab
```
Comment (lonico) [inline docs/source/install.rst]: Bitbucket

File: docs/source/release_notes.rst [hunk 4]
```diff
@@ -1,14 +1,19 @@
...
 ================================================================
 
-Release 1.1: Known Issues
+Release 1.2: Release Notes
+--------------------------------------
+
+* DevOps@Scale has been tested and validated to work with ONTAP NFS and iSCSI volumes
+* DevOps@Scale now supports deployment in a non-default Kubernetes NameSpace. Namespaces are intended for use in environments with many users spread across multiple teams, or projects
+* The solution now supports deployment across multiple storage classes. Each service has a customizable StorageClass field, which when not specified deploys the service with the Kubernetes default storage class
+* The solution now supports BitBucket SCM, in addition to GitLab. Please refer to Usage and setup in documentation section for detailed instructions.
```
Comment (lonico) [inline docs/source/release_notes.rst]: spelling: Bitbucket (here and below)

File: docs/source/usage.rst [hunk 5]
```diff
@@ -78,6 +78,48 @@ This will validate whether the GitLab and Jenkins integration has been successfu
       :width: 100%
       :alt: Successful build trigger on git push
 
+Integrate BitBucket with Jenkins for automatic build triggers
```
Comment (lonico) [inline docs/source/usage.rst]: spelling: Bitbucket (here and below)

**Comment by lonico** [comment]
@archanaganesan Is this PR still relevant, or superseded by the new one?

**Comment by archanaganesan** [comment]
> @archanaganesan Is this PR still relevant, or superseded by the new one?

Yes, I din't include these files in the other PR. Kept documentation separate

**Pull request merged**